http-server-netty/src/test/groovy/io/micronaut/http/server/netty/OptionsRequestAttributesSpec.groovy

```diff
@@ -37,6 +37,12 @@ class OptionsRequestAttributesSpec extends Specification {
         HttpClientResponseException e = thrown()
         e.response.status == HttpStatus.METHOD_NOT_ALLOWED
 
+        and:
+        MyFilter myFilter = ctx.getBean(MyFilter)
+        !myFilter.containsRouteInfo
```

Comment:
maybe also check that the filter was called at all

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/11097/commits/70d91bf05e49896caa6ab0735d430334f7afe051